tox.ini

```diff
@@ -4,10 +4,10 @@ envlist = py27
 [testenv]
 whitelist_externals = docker
 deps = -rrequirements.txt
-# auto parallel max b/c race condition with docker fixture (I think?)
+# 2 parallel max b/c race condition with docker fixture (I think?)
```

Comment:
My local > 2 core machine started failing more when I set this to auto.  I thought I had a parallel race condition fixed previously but there maybe more or perhaps it's related to the `time.sleep()` "fix"; slow to sync/write disks in docker's overlay2 FS.